jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxterrgv5.l

```diff
@@ -0,0 +1,205 @@
+;; vim: set ft=lisp:
+
+(require "package://baxtereus/baxter-util.l")
+(require "package://jsk_2016_01_baxter_apc/robots/baxterrgv5.l")
+
+(unless (find-package "JSK_2016_01_BAXTER_APC")
+  (make-package "JSK_2016_01_BAXTER_APC"))
+
+(defclass jsk_2016_01_baxter_apc::baxter-robot
```

Comment:
I know this is strange, but I have to overwrite `baxter-robot` class in https://github.com/start-jsk/jsk_apc/pull/2063/files#diff-953a5c100179528fcd593214aa478bb4R4 with `baxterrgv5` robot model for now.
So I named `baxterrgv5` robot model `baxter-robot` and didn't use `baxterrgv5` in euslisp codes.
In my thought, we can fix this when `baxterrgv5` robot model is merged into jsk_robot